com.unity.formats.fbx/Tests/FbxTests/FbxExportSettingsTest.cs

```diff
@@ -490,5 +491,21 @@ private void TestLocations(Dictionary<string,List<string>> data)
             }
         }
 
+        [Test]
+        public void TestFbxExportSettingsPreset()
+        {
+            var instance = ExportSettings.instance;
```

Comment:
I think you need to test what happens if you make a second export setting as well.